src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -614,7 +631,13 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
             size='lg'
             colorScheme='blue'
             onClick={handleSignTx}
-            isLoading={status === TxStatus.Pending || isInboundAddressLoading}
+            isDisabled={isTradingActive === false}
```

Comment:
This needs a copy for halted

Reply:
https://github.com/shapeshift/web/pull/6146/files#diff-29561149fe9fe8c5a89defe8be8fcfb91492b083ce61cf186b981069f24724c6R588